core/src/main/java/org/hisp/dhis/android/core/category/CategoryOptionComboCategoryLinkStoreImpl.java

```diff
@@ -56,9 +125,58 @@ private int executeInsert() {
         return lastId;
     }
 
-    @Override
-    public int delete() {
-        return databaseAdapter.delete(CategoryOptionComboCategoryLinkModel.TABLE);
+    private boolean execute(SQLiteStatement statement) {
+        int rowsAffected = databaseAdapter.executeUpdateDelete(CategoryComboModel.TABLE, statement);
+        statement.clearBindings();
+
+        return wasExecuted(rowsAffected);
+    }
+
+    private boolean wasExecuted(int numberOfRows) {
+        return numberOfRows >= 1;
+    }
+
+    private List<CategoryOptionComboCategoryLinkModel>
+    mapCategoryOptionComboCategoryLinksModelsFromCursor(
```

Comment:
this name is large, How only there is one method to map from cursor a name as mapFromCursor is more readable, can you rename it?

Reply:
Sure!